Livecheckables/dwarfutils.rb

```diff
@@ -1,6 +1,6 @@
 class Dwarfutils
   livecheck do
     url :homepage
-    regex(/HREF="libdwarf-([0-9.]+)\.t/i)
+    regex(/href=.*?libdwarf-([0-9.]+)\.t/i)
```

Comment:
```suggestion
    regex(%r{href=(?:["']?|.*?/)libdwarf-([0-9.]+)\.t}i)
```

There's a link to an alpha release with the format `alpha-libdwarf-20180706.tar.gz` that we should avoid. The relevant HTML looks like this:

```html
<A HREF="alpha-libdwarf-20180706.tar.gz" id="alpha-libdwarf-20180706">alpha-libdwarf-20180706.tar.gz</A>
```

This suggestion uses the typical snippet for `href` attributes that may only include the file name.